libs/styles/styles.css

```diff
@@ -155,6 +161,46 @@ a:hover {
   color: var(--link-hover-color);
 }
 
+/* Color scheme */
+.dark {
```

Comment:
I would sync this up / de-dupe with section-metadata. Having something as generic as `.dark` seems a bit scary as well. I'd rather define variables and let the blocks support this as they need. So `.marquee.dark` picks up the variables we define here. Same for section-metadata, z-pattern, etc.